spec/grape/integration/rack_spec.rb

```diff
@@ -22,7 +22,7 @@
       unless RUBY_PLATFORM == 'java'
         major, minor, patch = Rack.release.split('.').map(&:to_i)
         patch ||= 0 # rack <= 1.5.2 does not specify patch version
-        pending 'Rack 1.5.3 or 1.6.1 required' unless major >= 2 || (major >= 1 && ((minor == 5 && patch >= 3) || (minor >= 6)))
+        pending 'Rack 1.5.5 or 1.6.1 required' unless major >= 2 || (major >= 1 && ((minor == 5 && patch >= 5) || (minor >= 6)))
```

Comment:
this should not be changed, see #1066

Reply:
@darren987469 no worries, I addressed this in 87b6243f05b7dcb32431ad74528986499501164e